TASVideos/ViewComponents/PlayerPointsTable.cs

```diff
@@ -0,0 +1,53 @@
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using TASVideos.Core.Services;
+using TASVideos.Data;
+using TASVideos.Data.Entity;
+using TASVideos.WikiEngine;
+
+namespace TASVideos.ViewComponents
+{
+	[WikiModule(WikiModules.PlayerPointsTable)]
+	public class PlayerPointsTable : ViewComponent
+	{
+		private readonly ApplicationDbContext _db;
+		private readonly IPointsService _pointsService;
+
+		public PlayerPointsTable(ApplicationDbContext db, IPointsService pointsService)
+		{
+			_db = db;
+			_pointsService = pointsService;
+		}
+
+		public async Task<IViewComponentResult> InvokeAsync(int? count)
+		{
+			var showCount = count ?? 50;
```

Comment:
this might be avoidable by making the parameter of the invoke:  int? count = 50
Needs to be tested though, I'm not 100% sure it won't break the wiki module "model binder"

Reply:
I did try that before, but it just passes null if the parameter isn't passed through the module. That's why I did it like this.

Reply:
Yeah, that makes sense, was just an idea